.github/workflows/misc-linters.yml

```diff
@@ -28,3 +28,26 @@ jobs:
           pip install codespell
           # exclude files which may be synchronized from other places
           git ls-files | grep -v "^thirdparty" | grep -v "/thirdparty/" | grep -v "/dist/" | xargs -t codespell --ignore-words .github/workflows/ignore_words 2>/dev/null
+
+  linelint:
+    runs-on: ubuntu-latest
+    name: Check for missing EOF newlines
```

Comment:
You might missed this part from previous review, please change the job name to `linelint` here.